Tests/Application.UnitTests/Infrastructure/AppDbContextFactory.cs

```diff
@@ -20,6 +21,16 @@ public static ApplicationDbContext Create()
 
             context.Database.EnsureCreated();
 
+            var office = new Office {
```

Comment:
what about seeding our in-memory database and base our tests on that seed data?

Reply:
done